inspirehep/alembic/cb9f81e8251c_add_record_metadata_indices.py

```diff
@@ -30,7 +30,7 @@
 revision = 'cb9f81e8251c'
 down_revision = 'fddb3cfe7a9c'
 branch_labels = ()
-depends_on = None
+depends_on = '07fb52561c5c'
```

Comment:
this is needed because otherwise locally a migration trying to create an index on the `records_metadata.json` column using `jsonb` operators runs before the migration transforming that column to `jsonb`.